Marlin/temperature.cpp

```diff
@@ -782,6 +786,38 @@ static float analog2tempBed(int raw) {
   #endif
 }
 
+#ifdef MAX_THERMO_JUMP
+  static void check_max_thermo_jump(int8_t e) {
+    static bool init_jump[5] = {true, true, true, true, true};
+
+#ifdef HEATER_0_USES_MAX6675
+    if ((abs(((e < 0) ? current_temperature_bed_raw : current_temperature_raw[e]) - last_temperature_raw[e+1]) < (MAX_THERMO_JUMP_AMOUNT * ((e) ? OVERSAMPLENR : 1))) || init_jump[e+1]) {
+#else
+    if ((abs(((e < 0) ? current_temperature_bed_raw : current_temperature_raw[e]) - last_temperature_raw[e+1]) < MAX_THERMO_JUMP_AMOUNT * OVERSAMPLENR) || init_jump[e+1]) {
+#endif
+//      SERIAL_PROTOCOLPGM("jumptest: "); SERIAL_PROTOCOL(last_temperature_raw[e+1]); SERIAL_PROTOCOLPGM(":"); SERIAL_PROTOCOLLN(((e < 0) ? current_temperature_bed_raw : current_temperature_raw[e]));
+      last_temperature_raw[e+1] = ((e < 0) ? current_temperature_bed_raw : current_temperature_raw[e]);
+      init_jump[e+1] = false;
+      return;
+    }
+    else {
+//      SERIAL_PROTOCOLPGM("jumptest: "); SERIAL_PROTOCOL(last_temperature_raw[e+1]); SERIAL_PROTOCOLPGM(":"); SERIAL_PROTOCOLLN(((e < 0) ? current_temperature_bed_raw : current_temperature_raw[e]));
+
+// next is analogue to ( TRRunaway: ). kill() with adjustable message would be better
+      SERIAL_ERROR_START;
```

Comment:
Here I think you can use this:

    _temp_error(e, PSTR(MSG_THERMAL_JUMP_STOP), PSTR(MSG_ERR_THERMAL_JUMP));